galette/templates/default/elements/navigation/navigation_topbar.html.twig

```diff
@@ -1,9 +1,9 @@
 <header id="top-navbar" class="ui top fixed menu">
-    <div class="ui fluid container">
-        <a class="toc item" tabindex="0">
-            <i class="sidebar icon" aria-hidden="true"></i>
-            <span class="displaynone">{{ _T("Menu") }}</span>
-        </a>
+    <a class="toc item" tabindex="0">
+        <i class="sidebar icon" aria-hidden="true"></i>
+        <span class="visually-hidden">{{ _T("Menu") }}</span>
+    </a>
+    <nav class="ui fluid container" aria-label="{{ _T('Top menu') }}">
```

Comment:
I really do not like the label content. Also, semantically speaking, it is a menu on public pages, yes; but on logged in pages, it only is useful on mobile view, and is not in that cas really a menu but only a button to display/hide the menu.
I do not have propositions right now :/

Reply:
I don't really understand if your are talking about the "Menu" button or the "Top menu" `<nav>` element :thinking: 

**Regarding the "Menu" button**, it is outside the following `<nav>` element (which, indeed, is only displayed to unlogged users). So, it is simply a link to access the sidebar menu, and I think it is OK this way semantically speaking.

Moreover, when using the keyboard, this button is the second element in the tabulation key order, just after the "Skip to content" link. Which makes sense : if the user didn't wish to skip to content, it is logically to access the navigation.

**Regarding the "Top menu" `<nav>` element**, I suppose it could be OK if its `aria-label` is changed to "Main menu", and if I apply on it at the same time a `display: none` when the user is logged. This way in each case (mobile/desktop, un/logged, etc.), only one `nav` element labelled "Main menu" could be read by screen readers, as others are then hidden with `display: none` (this is only suppositions ; I am not a screen reader user nore an accessibility expert :wink:).

Otherwise, the mobile menu would require other improvements regarding accessibility using screen readers and/or the keyboard. For example, the text of the "Menu" button should actually be "Open the menu" by default, and changed to "Close the menu" when the menu is displayed. And I don't think that the fact that the button and the mobile menu are not children elements of the same parent is actually for screen readers.

But I don't want to go further regarding accessibility with screen readers for the moment. It seems to be a huge topic as far as I have read :sweat_smile:

Reply:
Thanks for the clarifications :)

I was talking about the "Top menu" label; that does not means anything really (and that also can be moved elsewhere in the future).
I guess we cannot use "Main menu" label once logged because the "real" "Main menu" can be displayed:
![image](https://github.com/galette/galette/assets/224733/341e7d8a-f7d0-40cb-a8b9-f7be3d0d55f3)

On the above screenshot, we can see it's a real menu only when user is not logged in; and therefore can be labelled "Main menu" whe no one is logged, and have no label once user has logged in (in facts, menu entries should not be rendered at all once logged in - but that is not a big issue).

For sure we have to limit the changes on accessibility (I was about to propose this one to be the very last before I release a RC3, and I'd like final release before the end of the month).

Reply:
> I guess we cannot use "Main menu" label once logged because the "real" "Main menu" can be displayed: ![image](https://user-images.githubusercontent.com/224733/284277680-341e7d8a-f7d0-40cb-a8b9-f7be3d0d55f3.png)
> 
> On the above screenshot, we can see it's a real menu only when user is not logged in; and therefore can be labelled "Main menu" whe no one is logged, and have no label once user has logged in (in facts, menu entries should not be rendered at all once logged in - but that is not a big issue).

I'll have a deeper look to this.

> For sure we have to limit the changes on accessibility (I was about to propose this one to be the very last before I release a RC3, 

For the final release, I think all recent changes were the best that could be done quite "easily", and were necessary or at least could benefit keyboard usability/accessibility :grin: 

> and I'd like final release before the end of the month).

Great ! :smiley:

Reply:
> > I guess we cannot use "Main menu" label once logged because the "real" "Main menu" can be displayed: ![image](https://user-images.githubusercontent.com/224733/284277680-341e7d8a-f7d0-40cb-a8b9-f7be3d0d55f3.png)
> > On the above screenshot, we can see it's a real menu only when user is not logged in; and therefore can be labelled "Main menu" whe no one is logged, and have no label once user has logged in (in facts, menu entries should not be rendered at all once logged in - but that is not a big issue).
> 
> I'll have a deeper look to this.

OK. I ended up with a simple solution in my upcoming changes : ensuring that _only one_ `<nav>` element is read by the screen reader when the user is unlogged or displaying the mobile view. This way, there is no real need (at the moment) to set a label to the involved `<nav>` elements in the templates.

Result with Orca when the user is unlogged in desktop view :
![image](https://github.com/galette/galette/assets/107203963/8cb109bc-8417-4f3c-a279-1f1cb0ddb136)

Result with Orca when the user is unlogged in mobile view (menu closed) :
![image](https://github.com/galette/galette/assets/107203963/fd553ad5-b922-455b-b7d5-2befca719e33)

Result with Orca when the user is unlogged in mobile view (menu opened) :
![image](https://github.com/galette/galette/assets/107203963/4cd08a55-d780-4ab8-8de5-4f58ec26f98c)

Then a label is set only when the user is logged in and not displaying the mobile view, because in this case the main navigation is "splited" across multiple `<nav>` elements.

Result with Orca when the user is logged :
![image](https://github.com/galette/galette/assets/107203963/61b4ce0f-0ff2-45c8-a7ac-f4de7df25e4c)

Reply:
OK, those results seems perfectly correct to me; thanks :)